snowfakery/cli.py

```diff
@@ -267,6 +283,18 @@ def validate_options(
         )
 
 
+def stopping_criteria_from_target_number(target_number):
+    "Deconstruct a tuple of 'str number' or 'number str' and make a StoppingCriteria"
+
+    # 'number str' is the official format so the other one can be deprecated one day.
```

Comment:
Isn't putting them in a canonical order already accomplished by int_string_tuple?

Reply:
I'll merge the embedding API next.

https://github.com/SFDO-Tooling/Snowfakery/compare/feature/embedding-api?expand=1